### PR TITLE
Sonnet V1.2.13

### PR DESCRIPTION
**Pull request by ultrabear**
will rc versions in a bit, someone should CR this but everything has been stable on dev for months (sans lib_datetimeplus, manualtests should be written)

### CODE REVIEW

**Comment by ultrabear** [comment]
tl;dr mariadb broke semver i think, 1.1.x requires a new clib version, so we need to use 1.0.x for 20.04

File: cmds/cmd_automod.py
Comment (bredo228) [inline cmds/cmd_automod.py:693]: versioning in a couple files is -DEV

Comment (bredo228) [reply cmds/cmd_automod.py:693]: don't know if this is intended or not

Comment (ultrabear) [reply cmds/cmd_automod.py:693]: yeah thats a TODO before i merge, same as the changelog

Comment (ultrabear) [reply cmds/cmd_automod.py:693]: will leave this convo open until ive pushed that

**Review by bredo228 — APPROVED** [review]
looks fine

File: cmds/cmd_automod.py
Comment (bredo228) [reply cmds/cmd_automod.py:693]: cool

**Comment by bredo228** [comment]
all good to merge once changelog / versioning sorted

**Pull request merged**
